src/EnergyPlus/HeatBalanceSurfaceManager.cc

```diff
@@ -441,30 +444,56 @@ namespace HeatBalanceSurfaceManager {
 			for ( SurfNum = 1; SurfNum <= TotSurfaces; ++SurfNum ) {
 				if ( Surface( SurfNum ).HasLinkedOutAirNode ) {
 					if ( Node( Surface( SurfNum ).LinkedOutAirNode ).OutAirDryBulbSchedNum != 0 ) {
-						Surface( SurfNum ).OutDryBulbTemp = GetCurrentScheduleValue( Node( Surface( SurfNum ).LinkedOutAirNode ).OutAirDryBulbSchedNum );
+						Surface( SurfNum ).OutDryBulbTemp = Node( Surface( SurfNum ).LinkedOutAirNode ).OutAirDryBulb;
```

Comment:
Do you still need to check for OutAirDryBuldSchedNum != 0 here (and 3 other places below)? If this LindedOutAirNode does not have a schedule then shouldn't Surface.OutDryBulbTemp = Node.OutAirDryBulb ?